logstash-core-plugin-api/lib/logstash-core-plugin-api/version.rb

```diff
@@ -1,2 +1,2 @@
 # encoding: utf-8
-LOGSTASH_CORE_PLUGIN_API = "2.1.7"
+LOGSTASH_CORE_PLUGIN_API = "2.1.12"
```

Comment:
Our git history sadly is lagging behind actual releases: https://rubygems.org/gems/logstash-core-plugin-api/versions/1.2.0-java